### DISCUSSION TITLE
記事に見せかけたアフィリエイトを行っていますが良いのでしょうか？

### DISCUSSION THREAD

**Discussion by tmkkz** [increments/qiita-discussions#539] [要望・アイデア・不具合]
https://qiita.com/Raio14f/items/96e02e97882515f277e8

こちらの記事でAmazonアフィリエイトを行っていますが、
Qiitaをアフィリエイト用プラットフォームとして使っても良いのでしょうか？

**Comment by tanaka-asami415** [comment]
@tmkkz 

大変恐れ入りますが、【初めにお読みください】 #116 にも記載しております通り
特定の記事やアカウントを記載する行為はお控えいただけますと幸いです🙇

> Qiitaをアフィリエイト用プラットフォームとして使っても良いのでしょうか？

Qiita利用規約では以下を定めております。



> （第11条第1項第1号）
本サービスが「エンジニアに関する知識を記録・共有する」サービスであることを認識し、これに該当する内容を投稿すること

また、以下の行為を禁止しております。

> （第11条第5項第4号）
客観的には事実上、広告・宣伝や商用を目的とした勧誘と認められる行為（検索サイト最適化又はアフィリエイトを目的とする投稿行為を含むがこれに限らない）。ただし、当社が本サービス上で別途定める場合はその限りではありません。

https://qiita.com/terms


エンジニアにとって役立つ情報であるかアフィリエイト目的か否かは、明確に線引きするのが難しい部分もあるため、運営による独断ではなくユーザーさまからの通報を元にコミュニティガイドラインに沿って対応を行っております。

気になる投稿がございましたら、[通報機能](https://help.qiita.com/ja/articles/qiita-report)からご報告いただけますと幸いです。
どうぞよろしくお願いいたします。
